### ISSUE TITLE
0.2.1 still requires options(sqlrendertempemulationschema = ) to be set on snowflake

### ISSUE THREAD

**Issue by DrTTrousers** [OHDSI/CohortGeneratorModule#39]
The most recent update has not abated the issue of needing to modify main.R with the line:

```
# limitations under the License.

options(sqlRenderTempEmulationSchema = "someschema")
# Module methods -------------------------
```

**Comment by anthonysena** [comment]
So we'd need to address this in Strategus so the option is properly passed. You can explicitly set the `tempEmulationSchema` when creating the CDM Execution Settings as described here: https://ohdsi.github.io/Strategus/reference/createCdmExecutionSettings.html. Could you give that a try? Or confirm that is set in your execution settings object?

**Comment by anthonysena** [comment]
I can confirm from the OHDSI Symposium that this is still an issue - the `tempEmulationSchema` should be passed here: https://github.com/OHDSI/CohortGeneratorModule/blob/main/Main.R#L52. A better fix will be to properly set this options() via Strategus.

**Comment by anthonysena** [comment]
Closed via OHDSI/Strategus#76
